src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form class="form">
+      <div
+        class="form__icon form__icon--big"
+        data-qa="big"
+      >
+        <input
+          class="input input__big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </div>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <div
+        class="form__icon form__icon--small"
+        data-qa="small"
+      >
+        <input
+          type="text"
+          placeholder="Try “Los Angeles“"
+          class="input input__small"
+        />
```

Comment:
The input elements are missing the 'name' attribute. This attribute is important for form submission and accessibility.